src/AtomsView.jl

```diff
@@ -1,12 +1,23 @@
 module AtomsView
+using ArgCheck
 using AtomsBase
 using Bio3DView
+using Colors
+using Makie
+using PeriodicTable
 using Unitful
-AbstractSystem = AtomsBase.AbstractSystem
+const AbstractSystem = AtomsBase.AbstractSystem
 
+export draw_system
+export draw_system!
 export visualize_structure
```

Comment:
I would drop this function and its dependencies completely. It was always intended to just have something ... not relying on JavaScript is the better solution in my opinion.